### PR TITLE
Change logic to frontend

### PR DESCRIPTION
**Pull request by puzzle-rusher**

### CODE REVIEW

**Comment by fluiderson** [comment]
You can as well update the code in accordance to the recently updated template.
- https://github.com/gear-dapps/app/pull/4

File: tests/state_tests.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+//
+//     let auction = init(&sys);
+//     sys.spend_blocks(DURATION);
+//
+//     if let StateReply::IsActive(is_active) = auction.meta_state(State::IsActive()).unwrap() {
+//         assert!(!is_active);
+//     } else {
+//         panic!("Can't get state");
+//     }
+// }
```
Comment (shamilsan) [inline tests/state_tests.rs:21]: Dead code?
```suggestion
```

**Review by shamilsan — APPROVED** [review]

File: tests/state_tests.rs (hunk 1 above)
Comment (puzzle-rusher) [reply tests/state_tests.rs:21]: No, it's not dead, but I can't test it right now because of gtest bug with time)

Comment (shamilsan) [reply tests/state_tests.rs:21]: What bug?

**Review by LouiseMedova — APPROVED** [review]

**Pull request merged**